ecom/src/app/dashboard/dashboard.component.html

```diff
@@ -1,2 +1,205 @@
-<app-header></app-header>
-<app-sidebar></app-sidebar>
\ No newline at end of file
+
+<div class="page-wrapper">
+  <div class="page-content">
+    <div class="row row-cols-1 row-cols-md-2 row-cols-xl-4">
+      <div class="col">
+        <div class="card radius-10 border-start border-0 border-3 border-info">
```

Comment:
as we already discussed please make comman card as well , so you do not neeed to add the HTML coe of card again and again.